src/components/sidebar-navigation/SmallScreenHeader.tsx

```diff
@@ -27,6 +28,9 @@ export const SmallScreenHeader = () => {
   const signedIn = useAuthStore((store) => store.signedIn);
   const { requestAuthentication } = useSignInRedirect();
 
+  const handleSignIn = () => {
+    requestAuthentication();
+  };
```

Comment:
Seems like this method is never in use

Reply:
https://github.com/near/near-discovery/pull/1205/files#diff-888d244b9fa1db25259a7e7653f15559c6774309abcc501903c0f69c5e2017bfR215